rfcs/0000-tsdb-dimensions.md

```diff
@@ -0,0 +1,240 @@
+
+
+Changes to :service mapping
+
+```yaml
+---
+- name: service
+  title: Service
+  group: 2
+  short: Fields describing the service for or from which the data was collected.
+  description: >
+    The service fields describe the service for or from which the data was collected.
+
+    These fields help you find and correlate logs for a specific
+    service and version.
+  footnote: >
+    The service fields may be self-nested under service.origin.* and service.target.*
+    to describe origin or target services in the context of incoming or outgoing requests,
+    respectively.
+    However, the fieldsets service.origin.* and service.target.* must not be confused with
+    the root service fieldset that is used to describe the actual service under observation.
+    The fieldset service.origin.* may only be used in the context of incoming requests or
+    events to describe the originating service of the request. The fieldset service.target.*
+    may only be used in the context of outgoing requests or events to describe the target
+    service of the request.
+  reusable:
+    top_level: true
+    expected:
+      - at: service
+        as: origin
+        beta: Reusing the `service` fields in this location is currently considered beta.
+        short_override: Describes the origin service in case of an incoming request or event.
+      - at: service
+        as: target
+        beta: Reusing the `service` fields in this location is currently considered beta.
+        short_override: Describes the target service in case of an outgoing request or event.
+  type: group
+  fields:
+
+    - name: address
+      level: extended
+      type: keyword
+      dimension: true
+      short: Address of this service.
+      description: >
+        Address where data about this service was collected from.
+
+        This should be a URI, network address (ipv4:port or [ipv6]:port) or a resource path (sockets).
+      example: 172.26.0.2:5432
+
+```
+Changes to host mapping
+
+```yaml
+---
+- name: host
+  title: Host
+  group: 2
+  short: Fields describing the relevant computing instance.
+  description: >
+    A host is defined as a general computing instance.
+
+    ECS host.* fields should be populated with details about the host on which
+    the event happened, or from which the measurement was taken.
+    Host types include hardware, virtual machines, Docker containers, and Kubernetes nodes.
+  type: group
+  fields:
+    - name: hostname
```

Comment:
Only spotted this now. Should we select `host.hostname` or `host.name` ?

Reply:
A related thread i found is here : https://github.com/elastic/observability-dev/issues/1056

Based on the inputs in the thread, `host.name` may be good option. I will change accordingly.